### PR TITLE
[LUPEYALPHA-934] FE LUP name change

### PR DESCRIPTION
**Pull request by alkesh**
new name for FE LUP: 
> Targeted retention incentive payments for FE teachers

### CODE REVIEW

File: app/views/further_education_payments/provider/claims/complete.html.erb
Comment (alkesh) [inline app/views/further_education_payments/provider/claims/complete.html.erb:1]: this page is not in the Figma

**Review by rjlynch — APPROVED** [review]
Looks good 👍

File: app/views/further_education_payments/claims/_ineligible_courses.html.erb [hunk 1]
```diff
@@ -5,11 +5,11 @@
     </h1>
 
     <p class="govuk-body">
-      In order to claim a levelling up premium payment, you must teach an <%= govuk_link_to "eligible FE course", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers#eligible-fe-courses", new_tab: true %>.
+    In order to claim a targeted retention incentive payment, you must teach an <%= govuk_link_to "eligible FE course", "#{t("further_education_payments.eligibility_criteria_link_url")}#eligible-fe-courses", new_tab: true %>.
     </p>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_courses.html.erb:12]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_fe_provider.html.erb [hunk 2]
```diff
@@ -28,7 +28,7 @@
     <%= govuk_button_link_to "Change FE provider", claim_path(current_journey_routing_name, "further-education-provision-search") %>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_fe_provider.html.erb:31]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_lack_teaching_responsibilities.html.erb [hunk 3]
```diff
@@ -5,11 +5,11 @@
     </h1>
 
     <p class="govuk-body">
-      In order to claim a levelling up premium payment, you must be employed as a member of staff with teaching responsibilities.
+      In order to claim a targeted retention incentive payment, you must be employed as a member of staff with teaching responsibilities.
     </p>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_lack_teaching_responsibilities.html.erb:12]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_lacks_teacher_qualification_or_enrolment.html.erb [hunk 4]
```diff
@@ -15,7 +15,7 @@
     ], type: :bullet %>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_lacks_teacher_qualification_or_enrolment.html.erb:18]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_less_than_half_hours_teaching_eligible_courses.html.erb [hunk 5]
```diff
@@ -5,11 +5,11 @@
     </h1>
 
     <p class="govuk-body">
-      In order to claim a financial incentive payment, at least half of your timetabled teaching hours should be spent teaching an <%= govuk_link_to "eligible FE course", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers#eligible-fe-courses", new_tab: true %>.
+    In order to claim a targeted retention incentive payment, at least half of your timetabled teaching hours should be spent teaching an <%= govuk_link_to "eligible FE course", "#{t("further_education_payments.eligibility_criteria_link_url")}#eligible-fe-courses", new_tab: true %>.
     </p>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_less_than_half_hours_teaching_eligible_courses.html.erb:12]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_must_at_least_half_hours_teaching_fe.html.erb [hunk 6]
```diff
@@ -14,7 +14,7 @@
     ], type: :bullet %>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_must_at_least_half_hours_teaching_fe.html.erb:17]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_must_be_recent_further_education_teacher.html.erb [hunk 7]
```diff
@@ -5,11 +5,11 @@
     </h1>
 
     <p class="govuk-body">
-      In order to claim a financial incentive payment, you must be in the first 5 years of your further education teaching career in England.
+      In order to claim a targeted retention incentive payment, you must be in the first 5 years of your further education teaching career in England.
     </p>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_must_be_recent_further_education_teacher.html.erb:12]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_must_teach_at_least_one_term.html.erb [hunk 8]
```diff
@@ -9,7 +9,7 @@
     </p>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_must_teach_at_least_one_term.html.erb:12]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_subject.html.erb [hunk 9]
```diff
@@ -5,11 +5,11 @@
     </h1>
 
     <p class="govuk-body">
-      In order to claim a levelling up premium payment, you must teach an <%= govuk_link_to "eligible FE subject", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers#eligible-fe-courses", new_tab: true %>.
+    In order to claim a targeted retention incentive payment, you must teach an <%= govuk_link_to "eligible FE subject", "#{t("further_education_payments.eligibility_criteria_link_url")}#eligible-fe-courses", new_tab: true %>.
     </p>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_subject.html.erb:12]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_subject_to_problematic_actions.html.erb [hunk 10]
```diff
@@ -15,7 +15,7 @@
     type: :bullet %>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_subject_to_problematic_actions.html.erb:18]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_teaching_less_than_2_5.html.erb [hunk 11]
```diff
@@ -5,11 +5,11 @@
     </h1>
 
     <p class="govuk-body">
-      In order to claim an incentive payment, you must be timetabled to teach at least 2.5 hours per week in the current term.
+      In order to claim a targeted retention incentive payment, you must be timetabled to teach at least 2.5 hours per week in the current term.
     </p>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_teaching_less_than_2_5.html.erb:12]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/_ineligible_teaching_less_than_2_5_next_term.html.erb [hunk 12]
```diff
@@ -9,7 +9,7 @@
     </p>
 
     <p class="govuk-body">
-      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+      For more information, check the eligibility criteria for <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/_ineligible_teaching_less_than_2_5_next_term.html.erb:12]:
```suggestion
      For more information, check the eligibility criteria for <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/claims/eligible.html.erb [hunk 13]
```diff
@@ -11,15 +11,15 @@
...
+        Based on what you’ve told us, you can apply for a targeted retention incentive payment of:
       </div>
 
       <div class="govuk-heading-l">
         <%= number_to_currency(@form.award_amount, precision: 0) %>
       </div>
 
       <div class="govuk-body">
-        For more information about why you are eligible, read about the <%= govuk_link_to "levelling up premium payments for early career further education teachers", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", new_tab: true %>.
+        For more information about why you are eligible, read about the <%= govuk_link_to "levelling up premium payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```
Comment (slorek) [inline app/views/further_education_payments/claims/eligible.html.erb:22]:
```suggestion
        For more information about why you are eligible, read about the <%= govuk_link_to "targeted retention incentive payments for early career further education teachers", t("further_education_payments.eligibility_criteria_link_url"), new_tab: true %>.
```

File: app/views/further_education_payments/landing_page.html.erb [hunk 14]
```diff
@@ -47,7 +47,7 @@
     ], type: :bullet %>
 
     <p class="govuk-body">
-      Further information about the eligibility criteria can be found in the <%= govuk_link_to "levelling up premium payments for FE teachers guidance", "https://www.gov.uk/guidance/levelling-up-premium-payments-for-fe-teachers", target: "_blank" %>.
+      Further information about the eligibility criteria can be found in the <%= govuk_link_to "levelling up premium payments for FE teachers guidance", t("further_education_payments.eligibility_criteria_link_url"), target: "_blank" %>.
```
Comment (slorek) [inline app/views/further_education_payments/landing_page.html.erb:50]:
```suggestion
      Further information about the eligibility criteria can be found in the <%= govuk_link_to "targeted retention incentive payments for FE teachers guidance", t("further_education_payments.eligibility_criteria_link_url"), target: "_blank" %>.
```

**Review by slorek — CHANGES_REQUESTED** [review]

**Review by slorek — APPROVED** [review]

**Pull request merged**